src/aap_eda/api/views/credential_type.py

```diff
@@ -122,6 +122,22 @@ def partial_update(self, request, pk):
                 {"errors": error}, status=status.HTTP_400_BAD_REQUEST
             )
 
+        if models.EdaCredential.objects.filter(
+            credential_type=credential_type
+        ).exists():
+            inputs = request.data.get("inputs")
+            injectors = request.data.get("injectors")
+            if inputs or injectors:
+                field = "inputs" if inputs else "injectors"
```

Comment:
What if both fields are changed? The message only says inputs.
Maybe we don't pick field here, simply says Modifications to inputs or injectors are not allowed